src/main/java/com/mpatric/mp3agic/AbstractID3v2Tag.java

```diff
@@ -958,9 +958,22 @@ public void setUrl(String url) {
 	}
 
 	@Override
-	public List<ID3v2ChapterFrameData> getChapters() {
+	@Deprecated
```

Comment:
Rationale for deprecating this method is that it couples the return-type to an implementation. Thus encouraging bad coding practices.

References:
[How and When to Dreprecate APIs](https://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html)  - Oracle's Java documentation